NGitLab.Tests/MergeRequest/MergeRequestClientTests.cs

```diff
@@ -25,17 +26,20 @@ public async Task Test_merge_request_api()
             await GitLabTestContext.RetryUntilAsync(
                     () => mergeRequestClient[mergeRequest.Iid],
                     mr => string.Equals(mr.MergeStatus, "can_be_merged", StringComparison.Ordinal),
-                    TimeSpan.FromSeconds(120))
+                    TimeSpan.FromSeconds(240))
```

Comment:
Retry for up to 4 minutes, really? I thought 2 minutes was already a bit long... unless perhaps we're waiting for the GitLab server to be up and running?

Reply:
I think I can revert this change as it doesn't help